cartridges/int_algolia/cartridge/scripts/algolia/model/algoliaLocalizedProduct.js

```diff
@@ -30,6 +30,28 @@ try {
 
 const ALGOLIA_IN_STOCK_THRESHOLD = algoliaData.getPreference('InStockThreshold');
 const INDEX_OUT_OF_STOCK = algoliaData.getPreference('IndexOutOfStock');
+const ATTRIBUTE_LIST = algoliaData.getSetOfArray('AdditionalAttributes');
+const stores = [];
+var SystemObjectMgr;
+var ProductInventoryMgr;
+
+if (ATTRIBUTE_LIST.indexOf('storeAvailability') !== -1) {
+    SystemObjectMgr = require('dw/object/SystemObjectMgr');
+    ProductInventoryMgr = require('dw/catalog/ProductInventoryMgr');
+    var storeIt = SystemObjectMgr.getAllSystemObjects('Store');
+    while (storeIt.hasNext()) {
+        var store = storeIt.next();
+        var storeInventoryListId = store.inventoryListID;
+        if (storeInventoryListId) {
+            var storeInventory = ProductInventoryMgr.getInventoryList(storeInventoryListId);
```

Comment:
You don't need that, you can access the inventoryList directly from the `Store` object: `store.inventoryList`

Reply:
good point! Refactored!